backend/app/controllers/application_controller.rb

```diff
@@ -2,4 +2,7 @@
 
 class ApplicationController < ActionController::Base
   # before_action :authenticate_user!
+
+  # this fixed user creation but I have no idea WHY
+  respond_to :html, :json
```

Comment:
@mattdelsordo What was the specific issue/test that this fixed? Seems like a client request was likely requesting something without sending the relevant JSON headers, or alternatively you needed one of the default Devise pages that does not have a JSON equivalent.

Reply:
I think it was probably the latter. All I remember is that I was having tons of issues with devise endpoints responding with html forms instead of json responses.